extensions/amp-ad-exit/0.1/test-e2e/test-amp-ad-exit.js

```diff
@@ -93,7 +91,10 @@ describes.endtoend(
       await expect(await controller.getCurrentUrl()).to.match(
         /^http:\/\/localhost:8000\/\?product2&r=0\.\d+$/
       );
-      await requestBank.withdraw('tracking');
+      const requests = await controller.networkLogger.getSentRequests(
+        'http://localhost:8000/amp4test/request-bank/e2e/deposit/tracking'
+      );
+      await expect(requests).to.have.length(1);
```

Comment:
This seems like something that can be reused by multiple test suites. Would it make sense to refactor it into a helper function?

Reply:
What do you have in mind? My idea was to keep `NetworkLogger`'s methods as simple as possible, like making sure assertion logic doesn't get mixed in, to give the test writer plenty of flexibility.

Reply:
Can we change the RequestBank implementation directly? Something like a `withdraw` function but is adaptable to accept an arbitrary URL.

Reply:
I was hoping to replace RequestBankE2E with this to avoid confusion with integration tests' RequestBank. What do you think?

Reply:
That would be great. Can we have a similar asynchronous await a request functionality?

```
await expect(request1);
await expect(request1);
await expect(request2);
```
would be more intuitive and less prone to error.

Reply:
Tweaked it slightly - how's this?
```
await expect(testUrl).to.be.sent;
```

Reply:
Nice idea to add the `sent` property. LG!

Reply:
Cool! I also added `await expect(testUrl).to.have.sendCount(n)`